docs/topics/images.rst

```diff
@@ -218,23 +228,23 @@ For example, to use the `<picture>` tag.
         {% image page.photo width-400 %}
     </picture>
 
-And to use the `<amp-img>` tag.
+And to use the ``<amp-img>`` tag (based on the `Mountains example <https://amp.dev/documentation/components/amp-img/#example:-specifying-a-fallback-image>`_ from the AMP docs):
 
 .. code-block:: html+django
 
-    {% image page.photo width-400 as photo %}
-    <amp-img src="{{ photo.url }}" width="{{ photo.width }}"
-        height="{{ photo.height }}" alt="{{ photo.alt }}">
-
-
-The ``attrs`` shortcut
------------------------
-
-You can also use the ``attrs`` property as a shorthand to output the attributes ``src``, ``width``, ``height`` and ``alt`` in one go:
-
-.. code-block:: html+django
-
-    <img {{ tmp_photo.attrs }} class="my-custom-class" />
+    {% image image width-550 format-webp as webp_image %}
```

Comment:
Is this conditional on #5249 getting merged? If so, perhaps we ought to stick with a jpeg-only example until we're confident that webp support will make it into 2.7.

Reply:
Let's do them both! @zerolab, can you make sure that #5249 happens?